src/main/java/edu/kis/vh/nursery/storage/IntArrayStack.java

```diff
@@ -0,0 +1,51 @@
+package edu.kis.vh.nursery.storage;
+
+public class IntArrayStack  implements IntStorageInterface 
+{
+	private static final int MAX_SIZE = 12;
+	private static final int ERROR = 0;
```

Comment:
ERROR: total variable initialization and validation should always remain -1.
The default return value should be moved to the interface (3.1.8).